server/app/auth/oidc/CustomOidcLogoutRequest.java

```diff
@@ -0,0 +1,87 @@
+package auth.oidc;
+
+import com.google.common.base.Strings;
+import com.nimbusds.openid.connect.sdk.LogoutRequest;
+import java.net.URI;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class CustomOidcLogoutRequest extends LogoutRequest {
```

Comment:
Not blocking: Consider marking this "final" to make the decision to subclass explicit rather than implicit.